idrstream/CP_Plugins/runcellpose.py

```diff
@@ -409,6 +409,9 @@ def run(self, workspace):
             channels = [0, 0]
 
         diam = self.expected_diameter.value if self.expected_diameter.value > 0 else None
+        
+        # convert float image data to int image data to remove segmentation differences between the image types

```

Comment:
Just for clarity, I think it is better to say that it is removing segmentation differences that are occurs between CP and DP runs. This is very small so feel free to leave it if you want.

Reply:
Done in https://github.com/WayScience/IDR_stream/pull/22/commits/d645dccb019e62c0528a334462516d4b114ca9d3!